src/main/java/dataProvider/ConfigFileReader.java

```diff
@@ -0,0 +1,76 @@
+package dataProvider;
+
+import enums.DriverType;
+
+import java.io.BufferedReader;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.Properties;
+
+public class ConfigFileReader {
+
+    private Properties properties;
+    private final String propertyFilePath = "src/test/resources/configs/configurations.properties";
+
+    public ConfigFileReader() {
+        BufferedReader bufferedReader;
+
+        try {
+            bufferedReader = new BufferedReader(new FileReader(propertyFilePath));
+            properties = new Properties();
+            try {
+                properties.load(bufferedReader);
+                bufferedReader.close();
+            } catch (IOException e) {
+                e.printStackTrace();
+            }
+        } catch (FileNotFoundException e) {
+            throw new RuntimeException("configurations.properties not found at " + propertyFilePath);
+        }
+    }
+
+    public String getDriverPath() {
+        String driverPath = properties.getProperty("driverPath");
+        if (driverPath != null) return driverPath;
```

Comment:
please format your code, do not ignore **if else** statement parenthesis

Reply:
Done